### PR TITLE
Added option -l to list upgradable packages

### PR DESCRIPTION
**Pull request by johannesst**
Added -l option to list upgradable packages after the performance data

### CODE REVIEW

File: plugins/check_apt_upgrade [hunk 1]
```diff
@@ -113,7 +120,7 @@ fi
 # Note that we use `"$@"' to let each command-line parameter expand to a
 # separate word. The quotes around `$@' are essential!
 # We need TEMP as the `eval set --' would nuke the return value of getopt.
-TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hnvt: -l skip-hwe-check -- "$@")
+TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hlnvt: -l skip-hwe-check -- "$@")
```
Comment (simondeziel) [inline plugins/check_apt_upgrade:123]:
```suggestion
TEMP=$($GETOPT -q -n "$SCRIPT_NAME" -o +hlnvt: -l skip-hwe-check -- "$@")
```
WDYT? `man 1 getopt` says `-s bash` is the default.

Comment (johannesst) [reply plugins/check_apt_upgrade:123]: I agree, but this is from your code. I will change this and upload with the merged shellcheck fixes

Comment (simondeziel) [reply plugins/check_apt_upgrade:123]: Right but since you changed the interpreter it makes sense to change it now, I think

Comment (johannesst) [reply plugins/check_apt_upgrade]: This was a misstake, I changed back to sh and readded the -s sh now

**Review by simondeziel — APPROVED** [review]

**Comment by simondeziel** [comment]
Would you mind squashing your commits into a single one, please?

File: plugins/check_apt_upgrade [hunk 2]
```diff
@@ -122,6 +129,7 @@ eval set -- "$TEMP"
 # Parse arguments
 while test "$1" != "--"; do case "$1" in
       -h) long_usage; ;;
+      -l) LIST_PACKAGES=$( "$BIN" "--package-names" 2>&1 > /dev/null ); shift;;
```
Comment (simondeziel) [inline plugins/check_apt_upgrade:132]: Isn't the `2>&1 > /dev/null` but getting rid of everything?

Comment (johannesst) [reply plugins/check_apt_upgrade:132]: This is actually needed since apt-check write the file names to stderr.

**Comment by johannesst** [comment]
I cleaned up everything, so should be good for merging

**Review by simondeziel — APPROVED** [review]

**Comment by simondeziel** [comment]
> I cleaned up everything, so should be good for merging

Thank you for your contributions.

**Comment by johannesst** [comment]
I have to thank for this plugin, this saved me the hassle do revinvent the wheel since check_apt is not working correctly :)

**Pull request merged**